.github/workflows/deploy_production.yml

```
@@ -125,6 +125,10 @@ jobs:
 
             - name: Deploy backend production CloudFormation stack
               run: pnpm --filter "antalmanac-cdk" backend-production deploy
+              env:
+                  GOOGLE_CLIENT_ID: ${{ secrets.GOOGLE_CLIENT_ID }}
+                  GOOGLE_CLIENT_SECRET: ${{ secrets.GOOGLE_CLIENT_SECRET }}
+                  GOOGLE_REDIRECT_URI: https://antalmanac.com
```

Shouldn't this be the `/auth` route?
```suggestion
                  GOOGLE_REDIRECT_URI: https://antalmanac.com/auth
```